src/katgpucbf/__init__.py

```diff
@@ -42,7 +42,7 @@
 DEFAULT_JONES_PER_BATCH: Final = 2**20
 DIG_HEAP_SAMPLES: Final = 4096
 DIG_SAMPLE_BITS: Final = 10
-ENGINE_DITHER_SEED_BITWIDTH: Final = 2**64
+ENGINE_DITHER_SEED_BITWIDTH: Final = 64
```

Comment:
Since you mention the line above, what do you think of renaming this to ENGINE_DITHER_SEED_BITS for consistency? It seems BITWIDTH is only used in one other place (tests/xbgpu/test_engine.py), so maybe just change that one to BITS too.